room/runtime/src/main/java/androidx/room/SQLiteCopyOpenHelper.java

```diff
@@ -217,4 +251,35 @@ private void copyDatabaseFile(File destinationFile) throws IOException {
                     + destinationFile.getAbsolutePath() + ").");
         }
     }
+
+    private SupportSQLiteOpenHelper createFrameworkOpenHelper() {
+        SupportSQLiteOpenHelper frameworkOpenHelper = null;
+        try {
+            FrameworkSQLiteOpenHelperFactory factory = new FrameworkSQLiteOpenHelperFactory();
+            Configuration configuration = Configuration.builder(mContext)
+                    .name(mDatabaseConfiguration.name)
+                    .callback(new Callback(mDatabaseVersion){
+                        @Override
+                        public void onCreate(@NonNull SupportSQLiteDatabase db) {
+                        }
+
+                        @Override
+                        public void onUpgrade(@NonNull SupportSQLiteDatabase db, int oldVersion,
+                                int newVersion) {
+                        }
+                    })
+                    .build();
+                frameworkOpenHelper = factory.create(configuration);
+        } catch (SQLiteException e) {
```

Comment:
I don't think we should catch all SQLiteException and ignore them.

Reply:
What do you think we should do with the exceptions here? Would it be enough to print some log here?

Reply:
We should let it propagate up and crash the app (or let the user of Room catch it), its the same as exceptions in `onOpen`, they would crash the app.